src/main/java/ch/njol/skript/conditions/CondIsStackable.java

```diff
@@ -0,0 +1,36 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+
+
+
+@Name("is Stackable")
+@Description("Check if item is stackable.")
+@Examples("%item% is stackable")
```

Comment:
```suggestion
@Examples("player's tool is stackable")
```
It's preferred to add more useful examples not just this